cmd/aws-vpc-cni/main.go

```diff
@@ -347,6 +349,17 @@ func validateEnvVars() bool {
 		}
 	}
 
+	// Validate that IP_COOLDOWN_PERIOD is a valid integer
+	ipCooldownPeriod, err, input := utils.GetIntFromStringEnvVar(envIPCooldownPeriod, defaultIPCooldownPeriod)
+	if err != nil {
+		log.Errorf("IP_COOLDOWN_PERIOD has to be a valid integer, user inputted: %v", input)
```

Comment:
nit: Let's combine this case and the case below, i.e. `if err != nil || ipCooldownPeriod < 0`.

Since this is a container log, let's change this to "IP_COOLDOWN_PERIOD MUST be a valid positive integer. %s is invalid"